apps/total-typescript/src/pages/tutorials/[module]/[lesson]/index.tsx

```diff
@@ -2,38 +2,36 @@ import React from 'react'
 import LessonTemplate from 'templates/lesson-template'
 import {GetServerSideProps} from 'next'
 import {getTutorial} from 'lib/tutorials'
-import {getBlockedLesson, getLesson} from 'lib/lessons'
+import {getLesson} from 'lib/lessons'
 import {checkIfConvertkitSubscriber} from '@skillrecordings/convertkit'
 
 export const getServerSideProps: GetServerSideProps = async (context) => {
   const {params} = context
   const lessonSlug = params?.lesson as string
   const subscriber = await checkIfConvertkitSubscriber(context)
 
-  const blockedLesson = await getBlockedLesson(lessonSlug)
+  const tutorial = await getTutorial(params?.module as string)
+  const lesson = await getLesson(lessonSlug)
```

Comment:
instead of loading the lesson twice I'm just stripping out the video property using the same logic below

I think tutorial and subscriber could be loaded client side, but we should also have subscriber as a cookie, so we can probably get it a lot faster than a CK round trip in cases where it's cached 🤔